pkg/core/bundle_list.go

```diff
@@ -12,52 +13,128 @@ import (
 )
 
 const (
-	maxBundlesToList = 1000000
+	maxBundlesToList  = 1000000
```

Comment:
naming conventions to mention, i'll start with the naming convention i added first:  this should've been `maxMetaFilesToProcess` or something similar.  it is _not_ the maximum number of bundles to list, and this name was based on my misunderstanding.  please choose a better name.

Reply:
ok will do